Makefile

```diff
@@ -19,7 +19,7 @@ LANGUAGE		?= java
 GRPCPLUGIN		?= /usr/bin/protoc-gen-grpc-$(LANGUAGE)
 
 PROTO_FILES		:= $(shell find stroeer -iname "*.proto" | sed "s/proto$$/$(TARGET_SUFFIX)/")
-PROTOC_VERSION	?= 3.19.4
+PROTOC_VERSION	?= 21.2
```

Comment:
Mittlerweile gibt es 21.5

Reply:
used largest common version between java and js which is `21.2`